packages/edgeidentity/js/models/AEPIdentityMap.js

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+
+@flow
+@format
+*/
+
+'use strict';
+
+class AEPIdentityMap {
+  isEmpty: boolean;
+  namespaces: string;
+  
+  constructor(isEmpty: boolean, namespaces: string) {
```

Comment:
This class needs some more work, might be good to use the default empty constructor and add functions for the rest, like addItem, removeItem ...